tests/common/plugins/fib.py

```diff
@@ -27,7 +27,7 @@ def announce_routes(ptfip, port, routes):
 def generate_routes(family, podset_number, tor_number, tor_subnet_number,
                     spine_asn, leaf_asn_start, tor_asn_start,
                     nexthop, nexthop_v6,
-                    tor_subnet_size = 128, max_tor_subnet_number = 16,
+                    tor_subnet_size, max_tor_subnet_number,
                     router_type = "leaf", tor_index=None):
```

Comment:
Method `generate_routes` is called in  `sonic-mgmt/tests/decap/test_decap.py`. Pls update it

Reply:
Updated method in decap case. Removed hardcoded values. TC tested and passed on t0 topo.